src/services/getExpectedApy.ts

```diff
@@ -11,16 +11,48 @@ export const getExpectedApy = (
   vt: number, // variable token balance
   margin: number, // margin until the current timestamp (margin + accrued cashflow)
```

Comment:
small comment, think worth renaming the margin variable to margin_plus_accrued_cf or smth along those lines otherwise hard to tell what margin is referring to in the body of the function

Reply:
ok